src/main/cms/cms_menu_firmware.c

```diff
@@ -25,6 +25,7 @@
 #include <ctype.h>
 
 #include <stdbool.h>
+#include <string.h>
```

Comment:
Pretty sure this isn't needed. :wink:

Reply:
It looks innocent, but is actually needed there. 

`cms_menu_firmware.c` makes use of `string.h`, but had no direct include. `string.h` was only indirectly included like this:
**string.h** -> **arm_math.h** -> **gyroanalyse.h** -> **gyro.h** -> **cms_menu_firmware.c**

After removing `arm_math.h` from `gyroanalyse.h`, there was a compilation error in `cms_menu_firmware.c`. That's why I included `string.h` directly to make `cms_menu_firmware.c` independent from other header files.